docs/user_guide/mitigation.rst

```diff
@@ -74,6 +74,62 @@ Preprocessing
    
 .. currentmodule:: fairlearn.preprocessing
 
+Preprocessing algortihms transform the dataset to mitigate possible unfairness
+present in the data.
+Preprocessing algorithms in Fairlearn follow the `scikit-learn transformers
+<https://scikit-learn.org/stable/modules/generated/sklearn.base.TransformerMixin.html#sklearn.base.TransformerMixin>`_
+class, meaning that they can :code:`fit` to the dataset and :code:`transform` it.
+A :code:`fit_transform` might not always be available, in which case the separate
+:code:`fit` and :code:`transform` methods need to be used in sequence.
+
+.. _correlation_remover:
+
+Correlation Remover
+~~~~~~~~~~~~~~~~~~~
+Sensitive features can be correlated with non sensitive features in the dataset.
+By applying the :code:`CorrelationRemover`, these correlations are filtered out while
+as much as possible information is retained (as measured by the least-squares error).
+This is mathemetically described as:
+
+.. math::
+        \min _{\mathbf{z}_{1}, \ldots, \mathbf{z}_{n}} \sum_{i=1}^{n}\left\|\mathbf{z}_{i}
+        -\mathbf{x}_{i}\right\|^{2} \\
+        \text{subject to} \\
+        \frac{1}{n} \sum_{i=1}^{n} \mathbf{z}_{i}\left(\mathbf{s}_{i}-\overline{\mathbf{s}}
+        \right)^{T}=\mathbf{0}
+
+In the example below, a subset of the `adult dataset <https://www.openml.org/d/1590>`_
```

Comment:
I'm not a huge fan of introducing another example with adult... I have a PR where one of the main blockers was that I am using adult. I feel that this should be a blocker for all PRs that introduce it.

Reply:
I can understand that! Is there an alternative dataset we are also fine with using, or is that still up for debate?

Reply:
It's one of the discussions that keep resurfacing... I'm afraid I don't have a great answer to this. Perhaps worth raising in the community call unless it's already been discussed there(?) @fairlearn/fairlearn-maintainers

Reply:
You should get unblocked once #1064 is resolved. Diabetes 130-hospitals should be more suitable.